pallets/ah-migrator/src/account.rs

```diff
@@ -257,15 +260,18 @@ impl<T: Config> crate::types::AhMigrationCheck for AccountsMigrator<T> {
 	/// the check that data has been correctly migrated to asset hub. It should also contain the
 	/// data previously stored in asset hub, allowing for more complex logical checks on the
 	/// migration outcome.
-	fn post_check(_rc_total_issuance_before: Self::RcPrePayload, _: Self::AhPrePayload) {
+	fn post_check(rc_pre_payload: Self::RcPrePayload, ah_pre_payload: Self::AhPrePayload) {
 		// Check that no failed accounts remain in storage
 		assert!(
 			RcAccounts::<T>::iter().next().is_none(),
 			"Failed accounts should not remain in storage after migration"
 		);
 
-		// TODO: Giuseppe @re-gius
-		//   run post migration sanity checks like:
-		//    - rc_migrated_out == ah_migrated_in - failed accounts
+		let (rc_checking_before, _) = rc_pre_payload;
+		let (ah_checking_before, _) = ah_pre_payload;
+
+		let ah_checking_after = <T as Config>::Currency::total_balance(&T::CheckingAccount::get());
+
+		assert_eq!(ah_checking_after, ah_checking_before + rc_checking_before, "Checking account balance on Asset Hub after migration should be the sum of the checking account balance before migration and the checking account balance on the relay chain before migration");
```

Comment:
nope, this is not correct. [look here](https://github.com/polkadot-fellows/runtimes/blob/f09750296a8ad40d64e295ff6568acac36c978f7/pallets/ah-migrator/src/account.rs#L200) for the explanation.